src/context/concrete_base.rs

```diff
@@ -210,25 +214,45 @@ impl ConcreteBaseAudioContext {
     /// The sample rate (in sample-frames per second) at which the `AudioContext` handles audio.
     #[allow(clippy::cast_precision_loss)]
     #[must_use]
-    pub fn sample_rate(&self) -> f32 {
+    pub(super) fn sample_rate(&self) -> f32 {
         self.inner.sample_rate.0 as f32
     }
 
     /// The raw sample rate of the `AudioContext` (which has more precision than the float
     /// [`sample_rate()`](BaseAudioContext::sample_rate) value).
     #[must_use]
-    pub fn sample_rate_raw(&self) -> SampleRate {
+    pub(super) fn sample_rate_raw(&self) -> SampleRate {
         self.inner.sample_rate
     }
 
+    /// This represents the number of seconds of processing latency incurred by
+    /// the `AudioContext` passing the audio from the `AudioDestinationNode`
+    /// to the audio subsystem.
+    // We don't do any buffering between rendering the audio and sending
+    // it to the audio subsystem, so this value is zero. (see Gecko)
+    #[allow(clippy::unused_self)]
+    #[must_use]
+    pub(super) const fn base_latency(&self) -> f64 {
```

Comment:
There's no need to put this method here, we can simply have it at the OnlineAudioContext